ocs_ci/deployment/vmware.py

```diff
@@ -1089,9 +1221,14 @@ def clone_openshift_installer():
     ocp_version = get_ocp_version()
     # supporting folder structure from ocp4.5
     if Version.coerce(ocp_version) >= Version.coerce("4.5"):
-        clone_repo(
-            constants.VSPHERE_INSTALLER_REPO, upi_repo_path, f"release-{ocp_version}"
-        )
+        if "sno" in config.ENV_DATA:
```

Comment:
we should have an explicit key for SNO in config.ENV_DATA dict

Reply:
Do you mean that the key should always exist? Like if it is not sno it should be False? Because now the key is created if the yaml have sno = True.